ocs_ci/helpers/helpers.py

```diff
@@ -29,10 +29,16 @@
     update_container_with_proxy_env,
 )
 from ocs_ci.ocs.utils import get_non_acm_cluster_config, get_pod_name_by_pattern
-from ocs_ci.ocs.utils import mirror_image
+from ocs_ci.ocs.utils import (
+    mirror_image,
+    get_expected_nb_db_psql_version,
+    get_nb_db_psql_version_from_image,
+    query_nb_db_psql_version,
+)
```

Comment:
could you combine these two import to one

Reply:
Fixed in https://github.com/red-hat-storage/ocs-ci/pull/10440/commits/8c529c662fb788c0869c34da40c3c25f02a4a0d2